packages/lit-hotkeys/tests/useHotkey.test.tsx

```
@@ -0,0 +1,231 @@
+// @vitest-environment happy-dom
+import { describe, expect, it, vi, beforeEach, afterEach } from 'vitest'
+import { renderHook, act } from '@testing-library/react'
```

ESLint

[error] Member 'act' of the import declaration should be sorted alphabetically.

(sort-imports)